espnet2/spk/encoder/conformer_encoder.py

```diff
@@ -33,20 +33,20 @@
     PositionwiseFeedForward,
 )
 from espnet.nets.pytorch_backend.transformer.repeat import repeat
+
+# TooShortUttError,; check_short_utt,
```

Comment:
@Jungjee, please confirm that they are not needed.

Reply:
This is okay.